pkg/admission/informers.go

```diff
@@ -40,17 +40,19 @@ type Informers struct {
 func NewInformers(kubeClient client.KubeClient, namespace string) *Informers {
 	stopChan := make(chan struct{})
 
-	informerFactory := informers.NewSharedInformerFactoryWithOptions(kubeClient.GetClientSet(), 0, informers.WithNamespace(namespace))
+	informerFactory := informers.NewSharedInformerFactoryWithOptions(
+		kubeClient.GetClientSet(),
+		0,
+		informers.WithNamespace(namespace),
+	)
 	informerFactory.Start(stopChan)
 
-	result := &Informers{
+	return &Informers{
```

Comment:
Also, please avoid changes which just reformats stuff. It's OK if it substantially improves readability, otherwise it's just distracting and can make backporting changes more difficult due to merge conflicts.

Reply:
ok, I see. I also ask about [chenyulin0719](https://github.com/chenyulin0719) about this issue, He told me that the way to solve this issue may be changing shim register configmap informer parameters. I will look into it.